src/content/en/2019/compression.md

```diff
@@ -131,61 +131,68 @@ Appoximately 38% of HTTP responses are delivered with text based compression. Th
 *   If you can support brotli and precompress resources, then compress to brotli level 11.  This is more computationally expensive than gzip - so precompression is an absolute must to avoid delays. 
 *   If you can support brotli and are unable to precompress, then compress to brotli level 5. This level will result in smaller payloads compared to gzip, with a similar computational overhead.
 
 
-# What Type of Content Are We Compressing
+## What Type of Content Are We Compressing?
 
-Most text based resources (such as HTML, CSS and JavaScript) can benefit from gzip and brotli compression. However, it’s often not necessary to use these compression techniques on binary resources, such as images, video and some web fonts because their file formats are already compressed.
+Most text based resources (such as HTML, CSS and JavaScript) can benefit from gzip or brotli compression. However, it’s often not necessary to use these compression techniques on binary resources, such as images, video and some web fonts because their file formats are already compressed.
 
-In the graph below, the top 25 content types are displayed with box sizes representing the relative amount of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded red, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded green to indicate that they are being compressed. However, the light green shading for some content types indicate that they are not compressed as consistently as the others. 
+In the graph below, the top 25 content types are displayed with box sizes representing the relative number of requests. The color of each box represents how many of these resources were served compressed. Most of the media content is shaded red, which is expected since gzip and brotli would have little to no benefit for them.  Most of the text content is shaded green to indicate that they are being compressed. However, the light green shading for some content types indicate that they are not compressed as consistently as the others. 
 
-![](../../../static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg)
+![Top 25 Compressed Content Types](../../../static/images/2019/15_Compression/ch15_fig2_compressed_content_types.jpg)
 
-Filtering out the 6 most popular content types allows us to see the rest of these content types more clearly.The application/json and image/svg+xml content types are compressed less than 65% of the time.   
+Filtering out the 7 most popular content types allows us to see the rest of these content types more clearly.
```

Comment:
Write out the number when fewer than 10 ([source](https://developers.google.com/style/numbers#numbers-as-words_1))
```suggestion
Filtering out the seven most popular content types allows us to see the rest of these content types more clearly.
```